### DISCUSSION TITLE
Help me changing monika's clothes!

### DISCUSSION THREAD

**Discussion by M0n1k4-dot** [Monika-After-Story/MonikaModDev#9861] [general]
I gifted her so many clothes and idk how to change herr clothes. I can only make her tie her hair with something else.. Help me

**Comment by Capricus02** [comment]
If you successfully gifted her the clothing it should be an option "Talk > 'Hey, Monika...' > Appearance > Can you change your clothes?"

If you don't have any options there or aren't seeing the clothing you gifted her, either:

1.) You did not gift her the clothing correctly (use [this link](https://github.com/Monika-After-Story/MonikaModDev/wiki/Gift-Giving-Guide) for how to gift properly, and bear in mind that Gift files must end in a .gift extension. The actual contents do not matter, only the filename does. Your OS might hide file extensions, in which case you will need to show file extensions in your file explorer to be able to change the extension).

or

2.) The type of clothing you got her may not be unlockable until you reach a certain affinity level.
